k8s/manifests/charts/ck-loadbalancer/templates/metallb/bgp-policy.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.bgp.enabled }}
+{{- if (eq .Values.driver "metallb") }}
+
+{{- range .Values.bgp.neighbors }}
+apiVersion: "metallb.io/v1beta2"
+kind: BGPPeer
+metadata:
+  name: {{ include "ck-loadbalancer.fullname" $ }}
```

Comment:
be consistent with the way you render this name. also, make sure to quote all string values (`| quote`) to be on the safe side